package.json

```diff
@@ -49,9 +49,15 @@
     "@types/react-slick": "^0.23.4",
     "@types/react-swipeable-views": "^0.13.0",
     "@types/supercluster": "^5.0.2",
+    "@types/yup": "^0.28.3",
     "@typescript-eslint/eslint-plugin": "^2.21.0",
     "@typescript-eslint/parser": "^2.21.0",
-    "apollo-boost": "^0.4.7",
+    "apollo-cache-inmemory": "^1.6.6",
+    "apollo-client": "^2.6.10",
+    "apollo-link": "^1.2.14",
+    "apollo-link-error": "^1.1.13",
+    "apollo-link-http": "^1.5.17",
+    "apollo-link-retry": "^2.2.16",
```

Comment:
I'm interested in knowing the benefits of replacing `apollo-boost`.

Reply:
It gives more control to how you can configure the `ApolloClient` instance. Needed here for the custom links.